README.md

```diff
@@ -48,6 +48,43 @@ $ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/rel
 $ oc apply -f bundle/manifests
 ```
 
+### Create Project
+
+Create the project on jira-service-desk with the following custom resource:
+
+```yaml
+apiVersion: jiraservicedesk.stakater.com/v1alpha1
+kind: Project
+metadata:
+  name: <Name>
+  namespace: default
+spec:
+  name: <ProjectName>
+  key: <ProjectKey>
+  projectTypeKey: <ProjectTypeKey>
+  projectTemplateKey: <ProjectTemplateKey>
+  description: <ProjectDescription>
+  assigneeType: <ProjectAssigneeType>
+  leadAccountId: <ProjectLeadAccountId>
+  url: <ProjectUrl>
+```
+
+### Add/Remove Customers to JSD Project
+
+Create a customer and than add/remove customer into jira-service desk project with the following custom resource
+
+```yaml
+apiVersion: jiraservicedesk.stakater.com/v1alpha1
+kind: Customer
+metadata:
+  name: <Name>
+spec:
+  name: <CustomerName>
+  email: <CustomerEmail>
+  projects:
+    - <AssociatedProjectKey>
+```
```

Comment:
As discussed we were supposed to mention limitations clearly in the documentation. Which is missing, please update this